ts/features/fims/history/store/reducer/index.ts

```diff
@@ -0,0 +1,46 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { getType } from "typesafe-actions";
+import { ConsentsResponseDTO } from "../../../../../../definitions/fims/ConsentsResponseDTO";
+import { FimsHistoryActions, fimsHistoryGet } from "../actions";
+
+export type FimsHistoryState = {
+  consentsList: pot.Pot<ConsentsResponseDTO, string>;
+};
+
+const INITIAL_STATE: FimsHistoryState = {
+  consentsList: pot.none
+};
+
+const reducer = (
+  state: FimsHistoryState = INITIAL_STATE,
+  action: FimsHistoryActions
+): FimsHistoryState => {
+  switch (action.type) {
+    case getType(fimsHistoryGet.request):
+      // first request for the history will reload the page
+      return action.payload.continuationToken
```

Comment:
Can't we just use `return { consentsList: pot.toLoading(state.consentsList) }`?

Reply:
welp yes, my idea was to allow for possible full reload this way, but I may be thinking too much ahead